main.py

```diff
@@ -0,0 +1,136 @@
+    ("HI", "Eng", "en"): "71de56dc-f83b-4899-93ab-7fe61e417c2e",
+    ("HI", "Wls", "en"): "71de56dc-f83b-4899-93ab-7fe61e417c2e",
+    ("HI", "Wls", "cy"): "1001ac43-093d-425c-ac7d-68df5147c603",
+    ("HI", "Nir", "en"): "ed1c2e9f-c81e-4cc2-889c-8e0fa1d2ce1b",
+    ("HI", "Nir", "ga"): "ed1c2e9f-c81e-4cc2-889c-8e0fa1d2ce1b",
+    ("HI", "Nir", "eo"): "ed1c2e9f-c81e-4cc2-889c-8e0fa1d2ce1b",
+    ("CE", "Eng", "en"): "4077d2cf-81cd-462d-9065-f227a7c39a8d",
+    ("CE", "Wls", "en"): "4077d2cf-81cd-462d-9065-f227a7c39a8d",
+    ("CE", "Wls", "cy"): "e4a4ebea-fcc8-463b-8686-5b8a7320f089",
+}
+
+
+# pylint: disable=too-many-return-statements
+def notify(request: Request) -> Tuple[str, int]:
+    if not request.method == "POST":
+        return "Method not allowed", 405
+
+    if not (data := request.json):
+        return "Missing notification request data", 422
+
+    if not (form_type := data.get("form_type")):
+        return "Missing form_type identifier", 422
+
+    if not (language_code := data.get("language_code")):
+        return "Missing language_code identifier", 422
+
+    if not (region_code := data.get("region_code")):
+        return "Missing region_code identifier", 422
+
+    template_id = template_id_mapping.get(
+        (form_type, language_code, region_code), os.getenv("NOTIFY_TEST_TEMPLATE_ID")
+    )
+
+    if not template_id:
+        return "No template id selected", 422
+
+    data_fields = ("email_address", "display_address", "tx_id", "questionnaire_id")
+    return send_email(
+        NOTIFY_API_KEY, {key: data.get(key) for key in data_fields}, template_id
+    )
+
+
+def send_email(api_key: str, data: Dict, template_id: str) -> Tuple[str, int]:
+    service_id = api_key[-73:-37]
+    api_key = api_key[-36:]
+
+    if not is_valid_uuid(service_id):
+        return "Service ID is not a valid uuid", 422
+
+    if not is_valid_uuid(api_key):
+        return "API key is not a valid uuid", 422
+
+    api_token = create_jwt_token(api_key, service_id)
+
+    session.headers.update(
+        {
+            "Content-type": "application/json",
+            "Authorization": f"Bearer {api_token}",
+            "User-agent": f"NOTIFY-API-PYTHON-CLIENT/{__version__}",
+        }
+    )
+
+    kwargs = json.dumps(
+        {
+            "template_id": template_id,
+            "personalisation": {"address": data["display_address"]},
+            "email_address": data["email_address"],
+        }
+    )
+
+    try:
+        response = session.post(
+            "https://api.notifications.service.gov.uk/v2/notifications/email",
+            kwargs,
+        )
+        response.raise_for_status()
+        entry = dict(
+            severity="INFO",
+            message="notify email requested",
+            tx_id=data.get("tx_id"),
+            questionnaire_id=data.get("questionnaire_id"),
+        )
+        print(json.dumps(entry))
+    except RequestException as error:
+        error_message = error.response.json()["errors"][0]
```

Comment:
Will `response` and `errors` always exist?

Reply:
Yes